SpaceInvaders/src/Controllers/BulletsController.java

```diff
@@ -0,0 +1,127 @@
+package Controllers;
+
+import GameObjects.Bullet;
+import GameObjects.EarthDefensiveBlock;
+import GameObjects.PlayerController;
+
+import java.util.ArrayList;
+
+public class BulletsController {
+    ArrayList<Bullet> bullets = new ArrayList<Bullet>();
+    public void CreateBullet(int xPos, int yPos, Boolean enemyBullet){
+        bullets.add(new Bullet(xPos, yPos, enemyBullet));
+    }
+
+    public int[][] GetBullets(int[][] battlefield){
+        bullets.forEach((b) -> {
+            battlefield[b.getPositionY()][b.getPositionX()] = 6;
+            if(b.getEnemyBullet() == true) battlefield[b.getPositionY() + 1][b.getPositionX()] = 6;
+            else battlefield[b.getPositionY() - 1][b.getPositionX()] = 6;
+        });
+        return battlefield;
+    }
+
+    public void MoveBullets(){

```

Comment:
( { ) turėtu buti naujoje eilutėje
public void MoveBullets(){
        for(int i = 0; i < bullets.size(); i++)
{
            if(bullets.get(i).getPositionY() < 17 && bullets.get(i).getEnemyBullet() == true) bullets.get(i).MoveBullet();
            else if (bullets.get(i).getEnemyBullet() == true) bullets.remove(i);
            else if(bullets.get(i).getPositionY() > 3 && bullets.get(i).getEnemyBullet() == false) bullets.get(i).MoveBullet();
            else if (bullets.get(i).getEnemyBullet() == false) bullets.remove(i);
        }
    }